conf/messages.en

```diff
@@ -452,11 +451,17 @@ view.create-case-confirmation.title=You''ve submitted your documents
 view.create-case-confirmation.heading=You''ve submitted your documents
 view.create-case-confirmation.reference=Your case reference number:
 view.create-case-confirmation.save.heading=Save your case reference number
-view.create-case-confirmation.save.description=You might need it if you contact us with any queries.
+view.create-case-confirmation.save.description=You will have to provide your case reference number if:
+view.create-case-confirmation.save.bullet1=you need to contact us with any queries
+view.create-case-confirmation.save.bullet2=we need further information from you
+view.create-case-confirmation.save.bullet3=you want to add more information or documents to this case
+view.create-case-confirmation.save.description2=Make a note or save a screenshot of your case reference number for your own records.
 view.create-case-confirmation.next.heading=Next steps
-view.create-case-confirmation.next.description=You will hear from us soon via CHIEF or your declaration software.
+view.create-case-confirmation.next.description=Once the document checks are complete, you will hear from us through CHIEF or your declaration software.
+view.create-case-confirmation.info.heading=If we need more information
+view.create-case-confirmation.info.description=We’ll contact you through CHIEF or your declaration software if we need more documents from you.
 view.create-case-confirmation.continue=Submit documents for a new case
-view.create-case-confirmation.footnote=View guidance on GOV.UK about <a href="#">pre-clearance checks</a> or <a href="#">C1601, C1602, C1603 forms</a>.
+view.create-case-confirmation.footnote=View guidance on GOV.UK about <a href="#">clearing goods entering, leaving or transiting the UK or EU</a> or <a href="#">C1601</a>, <a href="#">C1602</a>, <a href="#">C1603</a> forms.
```

Comment:
Even though it looks like they might be separate links (I thought this too at first) the C forms are all covered by one link 

We also need to capitalise the C in **clearing** goods